sources/host-ctr/go.mod

```diff
@@ -3,16 +3,16 @@ module host-ctr
 go 1.19
 
 require (
-	github.com/aws/aws-sdk-go v1.44.176
+	github.com/aws/aws-sdk-go v1.44.208
 	github.com/awslabs/amazon-ecr-containerd-resolver v0.0.0-20221221205310-1903c4ed45d1
 	github.com/containerd/containerd v1.6.15
```

Comment:
I think we should rebase this PR after https://github.com/bottlerocket-os/bottlerocket/pull/2825 gets merged so we can move to containerd 1.6.18 along with the other dep updates in one go.

Reply:
That's a good call - yes; if we're upgrading to containerd 1.6.18, we can wait for that to get in and rebase on that to upgrade those dependencies too